lib/committee/request_validator.rb

```diff
@@ -19,7 +19,7 @@ def call(request, data)
 
     def check_content_type!(request, data)
       if request.content_type && !empty_request?(request)
-        unless Rack::Mime.match?(@link.enc_type, request.content_type)
```

Comment:
arguments used conversely here.

this is definition of ```Rack::Mime.match?```
https://github.com/rack/rack/blob/7c574e2593922005ee7993c6aca93aa0383cd973/lib/rack/mime.rb#L28

first argument is expected to ```value```, second is expected to ```matcher```